javaagent-extension-api/src/main/java/io/opentelemetry/javaagent/extension/ConfigPropertiesUtil.java

```diff
@@ -1,40 +0,0 @@
-/*
- * Copyright The OpenTelemetry Authors
- * SPDX-License-Identifier: Apache-2.0
- */
-
-package io.opentelemetry.javaagent.extension;
-
-import io.opentelemetry.api.incubator.config.ConfigProvider;
-import io.opentelemetry.api.incubator.config.DeclarativeConfigProperties;
-import io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdk;
-import io.opentelemetry.sdk.autoconfigure.internal.AutoConfigureUtil;
-import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
-
-public class ConfigPropertiesUtil {
```

Comment:
@robsunday @jack-berg I removed this util class because the `AutoConfiguredOpenTelemetrySdk` instance of the agent returns the bridged `ConfigProperties` now